src/index.html

```diff
@@ -20,9 +20,9 @@ <h1>Leaderboard</h1>
     </div>
```

Comment:
:red_circle: [REQUIRED] Please make sure your web application work properly following the project requirements. Your refresh button is not working as expected Kindly implement the "Refresh" button that will get all scores for a game created by you from the API (receiving data from the API and parsing the JSON).

Reply:
> Correction

The logic has been modified so that the button works optimally.